keyboards/vn66/vn66.c

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2019 HnahKB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "vn66.h"
+
+void matrix_init_kb(void) {
+  led_init_ports();
```

Comment:
```suggestion
  led_init_ports();
  matrix_init_user();
```

The user function needs to be called here in case user keymaps do something in it :)